understanding/20/on-input.html

```diff
@@ -102,14 +102,14 @@ <h2>Examples of On Input</h2>
       <ul>
          
          <li>
-            A form is provided for creating calendar entries in a web based calendaring and scheduling
-            application. Along with the standard fields for subject, time and location, there
-            is a set of radio buttons to select the type of calendar entry to create.  The calendar
-            entry type can be meeting, appointment or reminder.  If the user selects the radio
-            for meeting, additional fields are displayed on the page for entering the meeting
-            participants. Different fields appear if the reminder button is chosen. Because only
-            parts of the entry change and the overall structure remains the same the basic context
-            remains for the user.  
+            A form is provided for creating calendar entries in a web-based calendaring and scheduling
+            application. Along with the standard fields for subject, time and location, 
+            a select dropdown allows the user to choose the type of calendar entry to create. The calendar
+            entry type can be meeting, appointment or reminder.  If the user selects the
+            for meeting option, additional fields are displayed on the page for entering the meeting
```

Comment:
"for" no longer makes sense after the switch from "selects the option for meeting" to "selects the for meeting option"

```suggestion
            meeting option, additional fields are displayed on the page for entering the meeting
```